### DISCUSSION TITLE
How to adjust decryption time in an already existing database file?

### DISCUSSION THREAD

**Discussion by KodeXall** [keepassxreboot/keepassxc#9183] [Q&A] [answered]
This answer right here

https://unix.stackexchange.com/a/712516

points out to an options tab no-longer existing in version **_2.7.4_**. Is the

_**Decryption Time**_

parameter available on the second window during creation of a brand new database the same parameter as the

**_Database > Database Settings... > Security > Security > Encryption Settings > Transform rounds_**

by any chance? If not- then where is it?

**Comment by droidmonkey** [comment, marked as answer]
Uncheck the "Advanced" checkbox in lower left corner of the database security settings window to see the simplified decryption time slider.

**Reply by KodeXall** [reply]
I see the _**Advanced**_ checkbox - but it is grayed out and I am unable to change it state

This is how it is in my old database and in any new tha I create

**Reply by droidmonkey** [reply]
To uncheck it you must be on the security tab of the database settings.

**Reply by KodeXall** [reply]
OK, now I have access to _**Decryption Time**_

But is there a sense of setting this option if after doing it I then go back to using _**Advanced Settings**_?

**Reply by droidmonkey** [reply]
User either or, not both
